types/global.d.ts

```diff
@@ -132,6 +132,8 @@ declare global {
     }
     interface Describe {
       jestPlaywrightSkip: JestParams<SkipOption>
+      jestPlaywrightDebug: JestPlaywrightTestDebug
+      jestPlaywrightConfig: JestPlaywrightTestConfig
```

Comment:
@mmarkelov is this correct? 
in https://github.com/playwright-community/jest-playwright/blob/master/src/extends.ts#L104 
I don't see describe being extended with `jestPlaywrightDebug` or `jestPlaywrightConfig`. Only `jestPlaywrightSkip`. 
Or am I missing something?

Reply:
@kenny-f my bad. I'll think about if we need these kind of helpers for `describe` or we just want to **skip** only

Reply:
@mmarkelov I think it's useful to have for `describe` as well as it allows people to group tests differently and not be restricted by what is extended and what isn't.